### PR TITLE
Fix redirection after action in search results

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  |no
| BC breaks?    |no
| Deprecations? |no
| Tests pass?   | yes
| Documentation |no
| Translation   | yes
| CHANGELOG.md  |no
| License       | MIT

Fix https://github.com/wallabag/wallabag/issues/7821

### CODE REVIEW

File: tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php [hunk 1]
```diff
@@ -1475,6 +1475,34 @@ public function testSearch()
...
+
+        $form = $crawler->filter('form[name=search]')->form();
+        $data = [
+            'search_entry[term]' => 'ActionInSearchResults',
+        ];
+
+        $crawler = $client->submit($form, $data);
+        $currentUrl = $client->getRequest()->getUri();
+        $element = $crawler->filter('a[data-action="delete"]')->link();
+        $crawler = $client->click($element);
```
Comment (yguedidi) [inline tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php:1500]: Maybe before the click you can assert on the value of the href attribute?

File: tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php [hunk 2]
```diff
@@ -1475,6 +1475,41 @@ public function testSearch()
         $this->assertCount(1, $crawler->filter($this->entryDataTestAttribute));
     }
 
+    /**
+     * @group plop
```
Comment (yguedidi) [inline tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php:1479]: why this?

File: tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php [hunk 3]
```diff
@@ -1475,6 +1475,41 @@ public function testSearch()
...
+     */
+    public function testActionInSearchResults()
+    {
+        $this->logInAs('admin');
+        $client = $this->getTestClient();
+
+        $config = $this->getLoggedInUser()->getConfig();
+        $config->setActionMarkAsRead(ConfigEntity::REDIRECT_TO_CURRENT_PAGE);
```
Comment (yguedidi) [inline tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php:1487]: ah this setting is used for delete action also?

**Review by yguedidi — CHANGES_REQUESTED** [review]

File: tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php (hunk 3 above)
Comment (nicosomb) [reply tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php:1484]: yes

**Review by yguedidi — APPROVED** [review]
Before merge you should squash the second commit, it doesn't make sense on it's own IMO

**Pull request merged**